[KOGITO-9095][KSW-Guides] Make Workflow Dev Profile builder image configurable via Platform (#338)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc

= Make Workflow Dev Profile builder image configurable via Platform
:compat-mode!:
// Metadata:
:description: Configuration workflow Dev Profile builder image via Platform
:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, config, openshift, containers

This document describes how to configure Workflow Dev Profile builder image via Platform with the {operator_name}.

> #### Override Builder image and version

Reviewer: ```suggestion
== Override Builder image and version
```

Author: ok

> By default the builder image use a stable version aligned to the Kogito version.

Reviewer: ```suggestion
By default the builder image use a stable version aligned to the {product_name} version.
```

Author: ok

> By default the builder image use a stable version aligned to the {product_name} version.

Reviewer 2: ```suggestion
By default the builder image uses a stable version aligned to the {product_name} version.
```

Author: ok

> In the dev mode you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:

Reviewer 3: ```suggestion
In the dev mode, you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:
```

> In the dev mode, you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:

Reviewer 2: ```suggestion
In the dev mode, you can override the default builder image using the `devBaseImage` in the KogitoServerlessPlatform:
```

Author: ok
